actionview/test/template/form_options_helper_test.rb

```diff
@@ -1510,42 +1510,65 @@ def test_grouped_collection_select_under_fields_for
 
   def test_weekday_options_for_select_with_no_params
     assert_dom_equal(
-      "<option value=\"Sunday\">Sunday</option>\n<option value=\"Monday\">Monday</option>\n<option value=\"Tuesday\">Tuesday</option>\n<option value=\"Wednesday\">Wednesday</option>\n<option value=\"Thursday\">Thursday</option>\n<option value=\"Friday\">Friday</option>\n<option value=\"Saturday\">Saturday</option>",
+      "<option value=\"Monday\">Monday</option>\n<option value=\"Tuesday\">Tuesday</option>\n<option value=\"Wednesday\">Wednesday</option>\n<option value=\"Thursday\">Thursday</option>\n<option value=\"Friday\">Friday</option>\n<option value=\"Saturday\">Saturday</option>\n<option value=\"Sunday\">Sunday</option>",
       weekday_options_for_select
     )
   end
 
   def test_weekday_options_for_select_with_index_as_value
     assert_dom_equal(
-      "<option value=\"0\">Sunday</option>\n<option value=\"1\">Monday</option>\n<option value=\"2\">Tuesday</option>\n<option value=\"3\">Wednesday</option>\n<option value=\"4\">Thursday</option>\n<option value=\"5\">Friday</option>\n<option value=\"6\">Saturday</option>",
+      "<option value=\"1\">Monday</option>\n<option value=\"2\">Tuesday</option>\n<option value=\"3\">Wednesday</option>\n<option value=\"4\">Thursday</option>\n<option value=\"5\">Friday</option>\n<option value=\"6\">Saturday</option>\n<option value=\"0\">Sunday</option>",
```

Comment:
@DRBragg @kaspth I find it odd that the default behavior changed to use Monday as the start of the week, but the index value still starts at 0 with Sunday 🤔

Reply:
It's is a little odd but since `Date.beginning_of_week` defaults to Monday in Rails it's what works best in a Rails context. If you set `config.beginning_of_week = :sunday` in your config file this will default to Sunday instead

Reply:
@DRBragg Right, but if `Date.beginning_of_week` defaults to Monday, that explains why "Monday" is the first `<option>` in the select but doesn't explain why "Sunday" has the 0 index, and "Monday" is 1. Is it a bug? 🤔

Reply:
Sunday is 0 because `I18n.translate("date.day_names")` returns an array where Sunday is the first item, `Date::DAYS_INTO_WEEK` returns a hash where the value of the key `:sunday` is 0, and in Ruby calling `wday` on a date that is a Sunday will return 0. I'm not 100% why `Date.beginning_of_week` defaults to Monday